kf4aic.c

```diff
@@ -186,8 +186,8 @@ EFI_STATUS start_systemd_boot(EFI_HANDLE image_handle)
 	return ret;
 }
 
-#ifdef USE_TRUSTY
 struct rot_data_t g_rot_data = {0};
```

Comment:
In general having such a global variable is a poor software architecture design.  Why can't you define an accessor interface, or have it at a local variable and then submit it to the right module ?

Reply:
Jeremy, i see your point. g_rot_data define here is just same with what we did in kernelfliner.c and kf4abl.c before. I agree global variable is not a good idea and at least it should be static one.  Xunou, before fix it, we need refine keymaster.c  which was using this g_rot_data

Reply:
Jeremy, please ignore this patch, i have created  to address global variable issue your mentioned
https://github.com/intel/kernelflinger/pull/37